doc/api/cli.markdown

```diff
@@ -0,0 +1,163 @@
+# Command Line Options
+
+<!--type=misc-->
+
+Node.js comes with a wide variety of CLI options. These options expose built-in
+debugging, multiple ways to execute scripts, and other helpful runtime options.
+
+To view this documentation as a manual page in your terminal, run `man node`.
+
+
+## Synopsis
+
+`node [options] [v8 options] [script.js | -e "script"] [arguments]`
+
+`node debug [script.js | -e "script" | <host>:<port>] …`
+
+`node --v8-options`
+
+Execute without arguments to start the [REPL][].
+
+_For more info about `node debug`, please see the [debugger][] documentation._
+
+
+## Options
+
+### `-v`, `--version`
+
+Print node's version.
+
+
+### `-h`, `--help`
+
+Print node command line options.
+The output of this option is less detailed than this document.
+
+
+### `-e`, `--eval "script"`
+
+Evaluate the following argument as JavaScript.
+
+
+### `-p`, `--print "script"`
+
+Identical to `-e` but prints the result.
+
+
+### `-c`, `--check`
+
+Syntax check the script without executing.
+
+
+### `-i`, `--interactive`
+
+Opens the REPL even if stdin does not appear to be a terminal.
+
+
+### `-r`, `--require module`
+
+Preload the specified module at startup.
+
+Follows `require()`'s module resolution
+rules. `module` may be either a path to a file, or a node module name.
+
+
+### `--no-deprecation`
+
+Silence deprecation warnings.
+
+
+### `--trace-deprecation`
+
+Print stack traces for deprecations.
```

Comment:
It's possible to understand what the flag does from this line - but I think that in the official docs it might be a good idea to give a little detail about it - namely that it prints stack trace for warnings logged to the console (and not the deprecations themselves).